torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -86,6 +91,7 @@ def freeze_bn_stats(self):
         return self
 
     def _forward(self, input):
+        assert isinstance(self.bn.running_var, torch.Tensor)
```

Comment:
I remembered there was an issue with this in `BatchNorm`, but here this is correct because `self.track_running_stats` is always True. So this is fine.